app/Handlers/Events/SendIncidentEmailNotificationHandler.php

```diff
@@ -69,6 +69,7 @@ public function handle(IncidentHasReportedEvent $event)
                 $mail = [
                     'email'           => $subscriber->email,
                     'subject'         => 'New incident reported.',
+                    'incident'        => $event->incident,
```

Comment:
@jbrooksuk I think this is not going to work since we send the mail to the queue and Laravel has to serialize it (problems). It would be better to decorate the component as well and push it into de view if exists and check if it's null or not. :)